python/python_frontend/translation.cpp

```diff
@@ -711,8 +711,11 @@ class FusionTranslator : public OptInConstDispatch {
 
     // The min and max reduction operations expect the dtype argument to by
```

Comment:
Noticed a pre-existing typo here
```suggestion
    // The min and max reduction operations expect the dtype argument to be
```